aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -2157,6 +2157,24 @@ TEST(OperatorRegistrationTest, getRegistrationsForDispatchKey) {
   ASSERT_TRUE(std::includes(all_ops.begin(), all_ops.end(), cpu_ops.begin(), cpu_ops.end(), cmp_lambda));
 }
 
+Tensor symint_op(Tensor self, c10::SymInt length) {
+  return self;
+}
```

Comment:
nit: return self.clone(), it's not good for non-compositeimplicitautograd operators to return self directly